src/struct/commands/CommandHandler.js

```diff
@@ -722,16 +722,16 @@ class CommandHandler extends AkairoHandler {
 
         if (!this.cooldowns.get(id)[command.id]) {
             this.cooldowns.get(id)[command.id] = {
-                timer: this.client.setTimeout(() => {
+                timer: setTimeout(() => {
                     if (this.cooldowns.get(id)[command.id]) {
-                        this.client.clearTimeout(this.cooldowns.get(id)[command.id].timer);
+                        clearTimeout(this.cooldowns.get(id)[command.id].timer);
                     }
                     this.cooldowns.get(id)[command.id] = null;
 
                     if (!Object.keys(this.cooldowns.get(id)).length) {
                         this.cooldowns.delete(id);
                     }
-                }, time),
+                }, time).unref(),
```

Comment:
Not sure if using the unref method here would be a great idea

As per the [Node.js docs](https://nodejs.org/api/timers.html#timers_timeout_unref)
> Calling `timeout.unref()` creates an internal timer that will wake the Node.js event loop. Creating too many of these can adversely impact performance of the Node.js application.

Reply:
Ok yeah I didn't understand that. This part confused me. I guess they ment calling it multiple times on the same timeout.
> Calling `timeout.unref()` multiple times will have no effect.

I'll removed it